quinn-proto/src/frame.rs

```diff
@@ -48,6 +48,7 @@ pub trait FrameStruct {
     const SIZE_BOUND: usize;
 }
 
+#[macro_export]
```

Comment:
This macro currently hardcodes special handling of transport protocol STREAM frames in the `Display` impl, which external code probably doesn't want. In lieu of a more abstract macro, I'd recommend not exporting this, and instead including a similar but distinct one in new crate.

Reply:
Yes, this module doesn't actually need to have use the same macro. Fixed.